src/ProcessedStory.ts

```diff
@@ -1,10 +1,8 @@
 import { z as zod } from 'zod';
 import type { StorybookStory } from './browser/StorybookPage';
+import type { AxeParams } from './index';
 
-type Params = {
-  skip: boolean;
-  disabledRules: string[];
-  timeout: number;
+type Params = AxeParams & {
```

Comment:
Would it make sense for the AxeParams to be specified here, and for `index.ts` to import the type? That way the type is defined where it's used?